lib/timers.js

```diff
@@ -496,7 +504,15 @@ exports.setInterval = function(callback, repeat, arg1, arg2, arg3) {
   return timeout;
 };
 
-exports.clearInterval = function(timer) {
+const clearInterval = exports.clearInterval = function(timer) {
```

Comment:
Why the addition of `clearInterval` variable? Doesn't seem used anywhere.

Reply:
was added in https://github.com/nodejs/node/commit/e32b9698fa324810bfe13501594b264eef694281 as a bug fix to previous iteration